lesson_19/vlad_butulan/context_managers.py

```diff
@@ -0,0 +1,57 @@
+"""
+Implement a context manager called just_some_exceptions that will handle KeyError, IndexError by printing a message and let any other exceptions propagate outside of the context manager.
+Implement the context manager by using both approaches: by using class and by using @contextmanager
+Write examples that prove the functionality for all the handled exceptions and for one exception that is not handled.
+"""
+import contextlib
+
+
+class Just_some_execeptions1():
+
+    def __init__(self, dct):
+        self.dct = dct
```

Comment:
it is interesting that you also made the dict in the context manager. It is not wrong but unexpected. Can you make the context manager without the dict and only to catch the exceptions if they occur. It is important to have the functions and classes as specialized as possible( do as little as possible)